cvastrophoto/rops/denoise/diffusion.py

```diff
@@ -0,0 +1,160 @@
+# cython: infer_types=True
+
+import PIL
+import math
+import numpy
+import cython
+import logging
+from skimage import color, measure
+from ..base import PerChannelRop
+if not cython.compiled:
+    globals().update(dict(fabs=abs))
+
+
+logger = logging.getLogger(__name__)
+
+class DiffusionRop(PerChannelRop):
+    def __init__(self,raw=None,T=0.0025,L=1.0/180,R=2,thr=3,steps=20,dt=1.0/7):
```

Comment:
Try to follor [PEP8 style](https://www.python.org/dev/peps/pep-0008/)

that means:

``` python
def __init__(self, raw=None, T=0.0025, L=1.0/180, R=2, thr=3, steps=20, dt=1.0/7):
```